src/main/java/com/mecatran/gtfsvtor/validation/triptimes/TooFastTravelValidator.java

```diff
@@ -134,9 +134,52 @@ private double getMaxSpeedMps(GtfsRoute route, ValidatorConfig config) {
 		return maxSpeedMps;
 	}
 
+	private int mapExtendedToBaseRouteType(int routeType) {
```

Comment:
We may want to move this method to GtfsRouteType? That would be easier to share, other validator may want to use this code.

Reply:
Agreed. Will move this.